apps/system/test/unit/notifications_test.js

```diff
@@ -144,6 +145,12 @@ suite('system/NotificationScreen >', function() {
     };
     navigator.mozL10n = MockL10n;
 
+    isDocumentHidden = false;
+    Object.defineProperty(document, 'hidden', {
+      configurable: true,
+      get: () => isDocumentHidden
```

Comment:
Should we do `set: (val) => isDocumentHidden = val` too for future proofing and consistency?